Engine/util/library_posix.h

```diff
@@ -48,8 +48,12 @@ class PosixLibrary : BaseLibrary
 
   AGS::Common::String BuildPath(char *path, AGS::Common::String libraryName)
   {
-    AGS::Common::String platformLibraryName = path;
-    platformLibraryName.Append("/lib");
+    AGS::Common::String platformLibraryName = "";
+    if (path) {
+      platformLibraryName = path;
```

Comment:
AGS has a very old code, which has several styles used by its author (and contributors).
Our coding convention: http://www.adventuregamestudio.co.uk/wiki/AGS_Engine_Coding_Conventions

If you check most of my additions for the last few years tried to correspond to that convention, except for cases when doing small fixes having different style in surrounding code.